### ISSUE TITLE
Settings - I get an "about:blank" page after navigating to the macOS link

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#37407]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Found when executing PR :** https://github.com/Expensify/App/pull/29991

**Version Number:** 1.4.44.0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Issue reported by:** Applause-Internal Team

## Action Performed:
1. Navigate to Settings - About - App download links - macOS
2. Tap on the "X" button

## Expected Result:
I should be returned to Expensify after tapping on the "X" buttor or I should be able to use the back button to navigate back.

## Actual Result:
I get an "about:blank" page after navigating to the macOS link. I can't use the browsers back button.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [x] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/Expensify/App/assets/115492554/dbaa3350-af45-4a4a-95a1-19fce65caf8d

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by izarutskaya** [comment]
@sakluger I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors.

**Comment by izarutskaya** [comment]
We think that this bug might be related to #wave8-collect-admins
CC @zanyrenney

**Comment by sakluger** [comment]
I agree this is a bug, since on the mobile app, we close the browser tab after clicking `x`.

I would say that this is `#vip-vsb`, I'll add to that dashboard as low priority.

**Comment by sakluger** [comment]
I'm going to go ahead and make this external - if it must be handled internally, let me know.

**Comment by allroundexperts** [comment]
@tienifr's [proposal](https://github.com/Expensify/App/issues/37407#issuecomment-1977956650) looks good to me. I think we should just focus on the bug on hand in this PR.

🎀 👀 🎀 C+ reviewed

**Comment by manishjain999** [comment]
When clicking the the link for macOS - the link is actually pointing to "NewExpensify.dmg" file and the mobile app tries to download that directly. The link should instead be to a download page URL like - https://use.expensify.com/download where the page opens and which then initiates a download on opening, then the "about:blank" issue should be resolved.

**Comment by lakchote** [comment]
@tienifr's proposal LGTM. Thanks @allroundexperts for reviewing.

**Comment by lakchote** [comment]
> ❌ There was an error making the offer to @tienifr for the Contributor role. The BZ member will need to manually hire the contributor.

@sakluger can you make the offer to the contributor? Thanks!

**Comment by sakluger** [comment]
Looks like we did send an offer to @tienifr (https://www.upwork.com/nx/wm/offer/101278408). So the automation worked but didn't realized it worked 😄.

**Comment by sakluger** [comment]
@allroundexperts could you please complete the BZ checklist?

@tienifr I sent you an offer through Upwork, please let me know when you've accepted it. Thanks!

**Comment by allroundexperts** [comment]
## Checklist
1. https://github.com/Expensify/App/pull/11110
2. https://github.com/Expensify/App/pull/11110/files#r1534762717
3. N/A
4. A regression test would be helpful here. The steps given in the OP look good enough to me.

**Comment by JmillsExpensify** [comment]
$500 approved for @allroundexperts
